srcpkgs/brother-brscan5/INSTALL

```diff
@@ -0,0 +1,9 @@
+case "${ACTION}" in
+    post)
+	# Add brother driver to sane configuration
+	_SANE_CONF='/etc/sane.d/dll.conf'
+	if [ "$(grep brother5 ${_SANE_CONF})" = '' ]; then
+		echo brother5 >> ${_SANE_CONF}
```

Comment:
Is there no other way to do this? It can't go into a different file?

Reply:
I copied the behavior of the Void brother-brscan4 package here. Personally I think a `/etc/sane.d/dll.d/` directory would be better, and this is what the Arch packages do, but I think we'd have to modify the sane packages to support this out of the box.

Is there a better way?

Reply:
Given `brother-brscan3` and `brother-brscan4` both manage entries in `/etc/sane.d/dll.conf` and that we'd likely have to modify other packages in order to change this behavior (i.e. `sane`) I think we should keep it as is.